src/libparms/db/parmsdb.cpp

```diff
@@ -440,8 +440,8 @@
     "DELETE FROM error "                     \
     "WHERE exitCode=?1;"
 
-#define DELETE_ALL_ERROR_BY_EXITCAUSE_REQUEST_ID "delete_error_by_exitcause"
-#define DELETE_ALL_ERROR_BY_EXITCAUSE_REQUEST \
+#define DELETE_ALL_ERROR_BY_ExitCauseREQUEST_ID "delete_error_by_exitcause"
```

Comment:
I confirmed that the issue is a side effect of the find and replace macro. I have made the necessary changes in commit [9b5fcc6](https://github.com/Infomaniak/desktop-kDrive/pull/174/commits/9b5fcc6302e43d6405e2e2366d637cbc585d9b2e).

Regarding the refactor of typedef and #define, even though it could fall within the scope of this PR, there are already a large number of modifications. Therefore, I prefer to address it in a separate PR. A ticket already exists to handle this code smell:
[Jira 🎫 ](https://infomaniak.atlassian.net/browse/KDESKTOP-739?atlOrigin=eyJpIjoiYjcwNDVjMjM0OGE1NDQ3NThhMzEyNjc5MWE2NDU2ZTciLCJwIjoiaiJ9)